### PR TITLE
README update

### PR DESCRIPTION
**Pull request by grumpycoder**
Updated readme to test PR.

### CODE REVIEW

**Comment by marklawrence-owl** [comment]
This looks good

**Review by marklawrence-owl — APPROVED** [review]
Approved

File: README.md
Comment (marklawrence-owl) [inline README.md:1]: Reviews readme update

**Review by marklawrence-owl — APPROVED** [review]

File: README.md
Comment (grumpycoder) [reply README.md:1]: thanks

**Pull request merged**